saas_public/doc/index.rst

```diff
@@ -0,0 +1,28 @@
+===============
+ SaaS Public
+===============
+
+Installation
+============
+
+* Follow instruction of `Job Queue <https://github.com/OCA/queue/tree/12.0/queue_job>`__ module.
+* `Install <https://odoo-development.readthedocs.io/en/latest/odoo/usage/install-module.html>`__ this module in a usual way
+* Restart the odoo as required by the Job Queue module
+
+Configuration
+=============
+
+* Open menu ``[[ SaaS ]] >> Templates``
+* Open your template or create new for which you need to create a build
+* Choose or create Template's deployments for which you can open public access
+* Check the box ``Public Access``
```

Comment:
Docs seems not updated after moving the field

Reply:
Use special formatting for checkboxes

Reply:
![image](https://user-images.githubusercontent.com/186131/60886366-32439100-a26b-11e9-8dd4-63c51a61d046.png)